ros_buildfarm/sourcedeb_job.py

```diff
@@ -116,6 +116,10 @@ def build_sourcedeb(sources_dir, os_name=None, os_code_name=None):
     cmd += [
         # dpkg-buildpackage args
         '-us', '-uc',
+        # set the option for dpkg-source to auto-commit upstream changes
+        # This is needed for Debian increments where the upstream has changed.
+        # It's not the best practice but we have people doing it a bunch.
+        '--hook-source=\'bash -c "echo auto-commit >> debian/source/options"\'',
```

Comment:
This only works when an existing file has a trailing newline. Please update it to also work when an existing file doesn't.